README.md

```diff
@@ -427,10 +427,94 @@ Called when the bounds in the calendar's canvas change. Use it for example to lo
+| `dimensions` | `object`    | returns the dimensions of the item which includes `collisionLeft`, `collisionWidth`, `height`, `isDragging`, `left`, `order`, `originalLeft`, `stack`, `top`, and `width`             |
+| `useResizeHandle`        | `boolean`    | returns the prop `useResizeHandle` from calendar root component                 |
+| `title`         | `string`    | returns title to render in content element.                    |
+| `canMove` | `boolean`    | returns if the item is movable.            |
+| `canResizeLeft`        | `boolean`    | returns if the item can resize from the left                 |
+| `canResizeRight`         | `boolean`    | returns if the item can resize from the right.                    |
+| `selected` | `boolean`    | returns if the item is selected.            |
+| `dragging`        | `boolean`    | returns if the item is being dragged                 |
+| `dragStart`         | `object`    | returns `x` and `y` of the start dragging point of the item.                    |
+| `dragGroupDelta`        | `number`    | returns number of groups the item moved. if negative, moving was to top. If positive, moving was to down                 |
+| `resizing`         | `boolean`    | returns if the item is being resized.                    |
+| `resizeEdge`        | `left`, `right`    | the side from which the component is being resized form                 |
+| `resizeStart`         | `number`    | returns the x value from where the component start moving                    |
+| `width`        | `boolean`    | returns the width of the item (same as in dimensions)                 |
+
+##### prop getters functions
+
+These functions are used to apply props to the elements that you render. This gives you maximum flexibility to render what, when, and wherever you like. 
+
+Rather than applying props on the element yourself and to avoid your props being overridden (or overriding the props returned). You can pass an object to the prop getters to avoid any problems. This object will only accept some properties that our component manage so the component make sure to combine them correctly.
+
+
+| property               | type              | description                                                                                 |
+| ---------------------- | ----------------- | ------------------------------------------------------------------------------------------- |
+| `getItemProps` | `function(props={})`    | returns the props you should apply to the root item element.            |
+| `getResizeProps`        | `function(props={})`    | returns two sets of props to apply on the `left` and `right` elements as resizing elements if you have `useResizeHandle` prop set to true                  |
+
+- `getItemProps` returns the props you should apply to the root item element. The returned props are:
+
+  - key: item id
+  - ref: function to get item referance
+  - className: classnames to be applied to the item
```

Comment:
Is className still needed here since you are using inline styles?

Reply:
yes it adds `rct-item`